src/configs/binary-elementwise-config.c

```diff
@@ -922,11 +922,19 @@ static void init_qs8_vadd_config(void) {
     qs8_vadd_config.init.qs8_add = xnn_init_qs8_add_minmax_scalar_params;
     qs8_vadd_config.element_tile = 4;
   #elif XNN_ARCH_RISCV
-    qs8_vadd_config.minmax.op_ukernel = (xnn_vbinary_ukernel_fn) xnn_qs8_vadd_minmax_ukernel__scalar_u4;
-    qs8_vadd_config.minmax.opc_ukernel = (xnn_vbinary_ukernel_fn) xnn_qs8_vaddc_minmax_ukernel__scalar_u4;
-    qs8_vadd_config.minmax.ropc_ukernel = (xnn_vbinary_ukernel_fn) xnn_qs8_vaddc_minmax_ukernel__scalar_u4;
-    qs8_vadd_config.init.qs8_add = xnn_init_qs8_add_minmax_scalar_params;
-    qs8_vadd_config.element_tile = 4;
+    #if XNN_ENABLE_RISCV_VECTOR
+      qs8_vadd_config.minmax.op_ukernel = (xnn_vbinary_ukernel_fn) xnn_qs8_vadd_minmax_ukernel__rvv_u2v;
+      qs8_vadd_config.minmax.opc_ukernel = (xnn_vbinary_ukernel_fn) xnn_qs8_vaddc_minmax_ukernel__rvv_u2v;
+      qs8_vadd_config.minmax.ropc_ukernel = (xnn_vbinary_ukernel_fn) xnn_qs8_vaddc_minmax_ukernel__rvv_u2v;
+      qs8_vadd_config.init.qs8_add = xnn_init_qs8_add_minmax_rvv_params;
+      qs8_vadd_config.element_tile = 2;
```

Comment:
Why do you set the element_tile size 2?
It means each time ukernel can only process 2 data.
Can we change to read vlenb and determine element_tile from vlenb & LMUL?